website/docs/sdk-reference/android.mdx

```diff
@@ -421,7 +421,7 @@ client.forceRefresh();
 
 With the following hooks you can subscribe to particular events fired by the SDK:
 
-- `onClientReady()`: This event is sent when the SDK reaches the ready state. If the SDK is configured with lazy load or manual polling it's considered ready right after instantiation.
+- `onClientReady(ClientReadyState)`: This event is sent when the SDK reaches the ready state. If the SDK is configured with lazy load or manual polling it's considered ready right after instantiation.
```

Comment:
In addition, [from what I see](https://github.com/configcat/android-sdk/blob/master/src/main/java/com/configcat/ConfigService.java#L94-L96), "with lazy load or manual polling it's considered ready right after instantiation" is no longer true.

If so, I suggest a wording like [this](https://github.com/configcat/docs/blob/master/website/docs/sdk-reference/dotnet.mdx?plain=1#L421).